packages/eslint-plugin-query/src/utils/create-rule.ts

```diff
@@ -3,7 +3,7 @@ import { detectTanstackQueryImports } from './detect-react-query-imports'
 import type { EnhancedCreate } from './detect-react-query-imports'
 
 const getDocsUrl = (ruleName: string): string =>
-  `https://tanstack.com/query/v4/docs/eslint/${ruleName}`
+  `https://tanstack.com/query/latest/docs/react/eslint/${ruleName}`
```

Comment:
urls have changed and are now:

https://tanstack.com/query/latest/docs/framework/react/eslint/stable-query-client

Reply:
Since these should actually be shown for all frameworks, I've moved them to: https://tanstack.com/query/latest/docs/eslint/stable-query-client

However, both of the above URLs would now correctly redirect thanks to changes in https://github.com/TanStack/tanstack.com